src/contrasts.jl

```diff
@@ -87,7 +87,7 @@ mutable struct MyCoding <: AbstractContrasts
 end
 
 contrasts_matrix(C::MyCoding, baseind, n) = ...
-termnames(C::MyCoding, levels, baseind) = ...
+_termnames(C::MyCoding, levels, baseind) = ...
```

Comment:
Isn't this breaking given that it's a documented function? It's kind of embarrassing given that it returns the names of generated columns/coefficients, not those of terms. Maybe we could rename it to something like `colnames` or `coefnames`, with a deprecation?

Reply:
Ugh, yeah I missed that it's in the [manual](https://juliastats.org/StatsModels.jl/stable/contrasts/#StatsModels.AbstractContrasts). Yes we should deprecate and make it a method of `coefnames`.